apps/prairielearn/src/lib/editors.ts

```diff
@@ -185,12 +185,19 @@ export function getUniqueNames({
  * than the default value.
```

Comment:
Nit, clarifying docs here:

```suggestion
 * than the default value.
 *
 * `defaultValue` may be either a value to compare directly with `===`, or a function
 * that accepts a value and returns a boolean to indicate if it should be considered
 * a default value.
```